extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -0,0 +1,269 @@
+            }
+
+            a:first-of-type:hover {
+                color: $color-links;
+                text-decoration: underline;
+                text-shadow: initial;
+            }
+
+            .title {
+                font-size: 12px;
+                height: 40px;
+                margin-top: 10px;
+
+                a {
+                    display: block;
+                    margin: 0;
+                    overflow: hidden;
+                    text-overflow: ellipsis;
+                    white-space: nowrap;
+                    word-wrap: normal;
+                }
+            }
+        }
+    }
+
+    &.new-chat {
+        .ChatModule {
+
+            @include wikia-rail-element;
+
+            p {
+                color: mix($color-text, $color-page, 60%);
+                font-size: 12px;
+            }
+
+            .chat-headline {
+
+                .chat-live {
+                    color: $color-text;
+                    font-size: 18px;
+                    font-weight: bold;
+                }
+
+                .chat-total {
+                    background-color: $color-rail-background;
+                    border: 1px solid;
+                    border-radius: 50%;
+                    font-size: 10px;
+                    height: 12px;
+                    margin: -5px 0 0 3px;
+                    padding: 2px;
+                    text-align: center;
+                    width: 12px;
+                }
+            }
+
+            .chat-join {
+                height: 42px;
+
+                button {
+                    background-image: none;
+                    border: 0;
+                    border-radius: 0;
+                    color: $white;
+                    height: 100%;
+                    text-transform: uppercase;
+                }
+
+                button:before {
+                    display: none;
+                }
+            }
+
+            a:hover {
+                text-decoration: none;
+            }
+
+            .avatar {
+                background-position: 50%;
+                border-radius: 50%;
+                height: 30px;
+                margin-right: 14px;
+                width: 30px;
+            }
+
+            .avatar:hover {
+                opacity: 1;
+            }
+
+            .chat-whos-here {
+                border: 0;
+                margin: 20px 0;
+                padding: 0;
+
+                .arrow-left {
+                    padding-left: 0;
+
+                    img {
+                        border-color: transparent;
+                        border-right-color: $color-links;
+                    }
+                }
+
+                .arrow-right {
+                    padding-right: 0;
+
+                    img {
+                        border-color: transparent;
+                        border-left-color: $color-links;
+                    }
+                }
+
+                .arrow-left.disabled, .arrow-right.disabled {
```

Comment:
This could be split into two lines.
I was also thinking about making a mixin out of this and then adding it with `&.disabled` in  the respective `.arrow-…` classes, but I'm not sure which is better. @vforge?